data_preparation/global_AgERA5/rename_agera5_files.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+
+# NOTE: AgERA5 indicators used in CY-Bench are:
+# Maximum_Temperature
+# Minimum_Temperature
+# Mean_Temperature
+# Precipitation_Flux
+# Solar_Radiation_Flux
+# See the AgERA5_params in download script.
+
+indicator = sys.argv[1]
+start_year = int(sys.argv[2])
+end_year = int(sys.argv[3])
+
+# rename files in the current directory
+# NOTE: CDS api seems to be changing.
+# From Sept 2024, this script may need an update.
```

Comment:
Is there a need to update this?

Reply:
Yes. There is no need to update the script. Removing comment.